packages/js/github-actions/actions/get-plugin-releases/action.yml

```
@@ -0,0 +1,24 @@
+name: Get Plugin L-2 release versions
+description: Get release versions for the L-2 releases from a specific Wordpress plugin like WooCommerce or Google Listing And Ads.
```

```suggestion
description: Get release versions for the L-x releases from a specific Wordpress plugin like WooCommerce or Google Listing And Ads.
```

It's not only for "L-**2**" :)